administrator/components/com_userlogs/views/userlogs/view.html.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_userlogs
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * View class for a list of logs.
+ *
+ * @since  1.6
+ */
+class UserlogsViewUserlogs extends JViewLegacy
+{
+    /**
+	 * An array of items
+	 *
+	 * @var  array
+	 */
+	protected $items;
+
+    protected $state;
+    /**
+     * Method to display the view.
+     *
+     * @param   string  $tpl  A template file to load. [optional]
+     *
+     * @return  void
+     *
+     * @since   3.6
+     */
+    public function display($tpl = null)
+    {
+        $this->items     = $this->get('Items');
+        $this->state     = $this->get('State');
+        $this->filterForm    = $this->get('FilterForm');
+        $this->activeFilters = $this->get('ActiveFilters');
+        if (count($errors = $this->get('Errors')))
+		{
+			JError::raiseError(500, implode("\n", $errors));
```

Comment:
Don't use JError any more, use Exceptions directly.

throw new ..

(Yeah error handling in joomla is generally not the yellow from the egg..).